lib/liftoff/xcodeproj_helper.rb

```diff
@@ -53,18 +53,20 @@ def add_script_phases(scripts)
       end
     end
 
-    def perform_extra_config(extra_config)
-      if extra_config
-        extra_config.each do |name, settings|
-          if name.downcase == "all"
-            object = target
-          else
-            object = target.build_settings(name)
-          end
+    def perform_extra_config(app_config, test_config)
+      [app_config, test_config].each do |config|
+        if config
+          config.each do |name, settings|
+            if name.downcase == "all"
+              object = target
+            else
+              object = target.build_settings(name)
+            end
 
-          if object
-            settings.each do |key, value|
-              object[key] = value
+            if object
+              settings.each do |key, value|
```

Comment:
so if `object` is a specific build setting, then it _is_ a hash. But if it's the target itself, it isn't a hash. This is probably actually a mess I got myself into by trying to simplify this code.